source/brsHamcrest_Helpers.brs

```diff
@@ -109,3 +109,34 @@ function coreDoMatch (target as Dynamic, comparison as Dynamic) as Boolean
 
     return true
 end function
+
+
+'Return a normalised type-value to handle the fact that calling type(target) can vary in the actual type-value returned.
+'
+'@param typeIn {String} the type-value to normalise
+'@return {String} the normalised type-value
+function BrsHamcrestNormaliseType (typeIn as String) as String
```

Comment:
What happens when an Invalid type is passed in?

Reply:
Updated to trigger BrsHamcrest error and return `"<ERROR:UNSUPPORTED_TYPE>"`.